docs/node-mixin/dashboards/node.libsonnet

```
@@ -224,6 +281,30 @@ local gauge = promgrafonnet.gauge;
           type: 'datasource',
         },
       )
+      .addTemplate(
+        {
+          current: {
+            text: 'Loki',
+            value: 'Loki',
+          },
+          hide: 0,
+          label: 'Loki Data Source',
+          name: 'logs_datasource',
```

We're stumbling toward a consensus that a dashboard with multiple datasources should name them unambiguously, with their type. I.E. This should be..

```suggestion
          name: 'loki_datasource',
```

Further, the metrics datasource should be renamed `prometheus_datasource`.